internal/test/cleanup/cleanup.go

```diff
@@ -189,3 +195,73 @@ func NutanixTestResources(clusterName, endpoint, port string, insecure, ignoreEr
 	}
 	return nil
 }
+
+// TinkerbellTestResources cleans up machines by powering them down.
+func TinkerbellTestResources(ctx context.Context, inventoryCSVFilePath string, ignoreErrors bool) error {
```

Comment:
You moved other code from `vms` to `machines`. This function defined `resources`. Are resources different than machines?

Reply:
In this specific instance, `TinkerbellTestResources` is intended to handle machines. I named it this way to align with existing method of naming these cleanups. The term "resources" was used more broadly to encompass potential future expansions where we might manage other types of test resources, not just machines. However, I see how this could lead to confusion given our current usage. I'll rename this to `TinkerbellTestMachines` and we could always expand around it if needed.